mteb/tasks/STS/ind/STSIDMT.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+from datasets import Value
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskSTS import AbsTaskSTS
+
+_EVAL_SPLIT = "test"
+
+
+class STSIDMT(AbsTaskSTS):
+    metadata = TaskMetadata(
+        name="STSIDMT",
+        dataset={
+            "path": "LazarusNLP/stsb_mt_id",
+            "revision": "d0095a14faeaad923263a0ce2d301f8163b94106",
+            "trust_remote_code": True,
+        },
+        description="Indonesian test sets from STS-B translated using Google Translate API",
+        reference="https://huggingface.co/datasets/LazarusNLP/stsb_mt_id",
+        type="STS",
+        category="s2s",
+        eval_splits=[_EVAL_SPLIT],
+        eval_langs=["ind-Latn"],
+        main_score="cosine_spearman",
+        date=("2012-01-01", "2024-01-04"),  # rough estimate,
+        form=["spoken", "written"],
+        domains=["News", "Social", "Web"],
+        task_subtypes=[],
+        license="apache-2.0",
+        socioeconomic_status="mixed",
+        annotations_creators="human-annotated",
+        dialect=[],
+        text_creation="machine-translated",
```

Comment:
We do not accept machine-translated datasets unless human-verified (with few exceptions).

Reply:
Note that if you e.g. want to create an Indonesian benchmark and find that a translation STS task is required we have had previous cases where we have done it, but generally we have found that machine translated datasets are not required and likely contains unknown biases.

Reply:
Thanks @KennethEnevoldsen, No problem!